src/Calculator/Resources/en-US/Resources.resw

```diff
@@ -4469,6 +4469,50 @@
   <data name="selectAllEquationMenuItem.Text" xml:space="preserve">
     <value>Select All</value>
     <comment>Select all menu item from the Equation TextBox</comment>
+  </data>    
+  <data name="EquationInputButton.[using:Windows.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">
+    <value>Function Input List Item</value>
+    <comment>The automation name for the Equation Input ListView item that is shown when Calculator is in graphing mode.</comment>
+  </data>
+  <data name="EquationInputList.[using:Windows.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">
+    <value>Function Input List</value>
+    <comment>The automation name for the Equation Input ListView that is shown when Calculator is in graphing mode.</comment>
+  </data>
+  <data name="EquationInputPanel.[using:Windows.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">
+    <value>Function Input Panel</value>
+    <comment>The automation name for the Equation Input StackPanel that is shown when Calculator is in graphing mode.</comment>
+  </data>
+  <data name="VariableStackPanel.[using:Windows.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">
+    <value>Variable Panel</value>
+    <comment>The automation name for the Variable StackPanel that is shown when Calculator is in graphing mode.</comment>
+  </data>
+  <data name="VariableListView.[using:Windows.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">
+    <value>Variable List</value>
+    <comment>The automation name for the Variable ListView that is shown when Calculator is in graphing mode.</comment>
+  </data>
+  <data name="VariableListViewItem.[using:Windows.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">
+    <value>Variable List Item</value>
```

Comment:
It might nice for narrator to read the the name of the variable. Something like "Variable M List Item". But we can possibly do that in a different PR.